steam.py

```diff
@@ -1,5 +1,8 @@
+from genericpath import exists
+from pickle import TRUE

```

Comment:
Uneeded imports, you're probably using the wrong true

Reply:
no u